packages/client/src/__tests__/integration/happy/exhaustive-schema-mongo/__snapshots__/test.ts.snap

```diff
@@ -2191,6 +2191,22 @@ export namespace Prisma {
       args: SelectSubset<T, PostFindUniqueArgs>
     ): HasReject<GlobalRejectSettings, LocalRejectSettings, 'findUnique', 'Post'> extends True ? Prisma__PostClient<PostGetPayload<T>> : Prisma__PostClient<PostGetPayload<T> | null, null>
 
+    /**
+     * Find one Post that matches the filter or throw
```

Comment:
Might be a good idea to update jsdoc to refer to the error code instead of deprecated class. Not critical though, at least not until class gets removed.

Reply:
👍🏾 Addressed in [64b0f70](https://github.com/prisma/prisma/pull/16162/commits/64b0f7057b3c03929a5c13bc74588381ec8f9a7d)